partner_communication_revision/data/demo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="template" model="mail.template">
+            <field name="name">Donation - Thank You Letter</field>
+            <field name="model_id" ref="partner_communication.model_partner_communication_job"/>
+            <field name="use_default_to" eval="True"/>
+            <field name="subject">Donation Receipt</field>
+            <field name="body_html" type="html">
+                <div>
+                    % set invoice_lines = object.get_objects()
+                </div>
+                <div>
+                    ${partner.salutation},
```

Comment:
This will fail when generating a letter because `partner` is not a variable declared in the template. Please correct it because some people may use the demo data to try the functionality of the module in an empty database instance, so we want this to work.